app/services/shopify_import/products/create.rb

```diff
@@ -0,0 +1,46 @@
+module ShopifyImport
+  module Products
+    class Create
+      def initialize(shopify_data_feed)
+        @shopify_data_feed = shopify_data_feed
+      end
+
+      def save!
+        Spree::Product.transaction do
+          @spree_product = create_spree_product
+          assign_data_feed_to_product
+          add_tags
+        end
+      end
+
+      private
+
+      def create_spree_product
+        Spree::Product.create!(product_attributes)
+      end
+
+      def assign_data_feed_to_product
```

Comment:
I'd call this method `assign_spree_product_to_data_feed` or similar. We're not adding data_feed to a product.